pkgs/applications/networking/instant-messengers/rambox/default.nix

```diff
@@ -32,6 +48,9 @@ stdenv.mkDerivation rec {
     mkdir -p $out/bin $out/share/rambox
     cp -r * $out/share/rambox
     ln -s $out/share/rambox/Rambox $out/bin
+
+    mkdir -p $out/share/applications
+    ln -s ${desktopItem}/share/applications/* $out/share/applications
```

Comment:
@Mic92 I was inspired by
https://github.com/NixOS/nixpkgs/blob/354fd3728952c229fee4f2924737c601d7ab4725/pkgs/games/orthorobot/default.nix#L15
https://github.com/NixOS/nixpkgs/blob/354fd3728952c229fee4f2924737c601d7ab4725/pkgs/games/orthorobot/default.nix#L52
In this case path to icon in desktop file will be path to downloaded icon in nix store. Or this is not a good way to do things?